assets/style/services/services.scss

```
@@ -5,3 +5,329 @@
+    @media (max-width: 1400px) {
+      padding: 40px;
+      padding-top: 60px;
+    }
+    @media (max-width: 768px) {
+      max-width: 100%;
+    }
+    @media (max-width: 480px) {
+      padding: 10px;
+    }
+  }
+}
+.main__drilling-work-foto2 {
+  @media (max-width: 1400px) {
+    order: -1;
+  }
+  img {
+    padding: 80px 0px;
+    @media (max-width: 1400px) {
+      padding: 40px;
+    }
+    @media (max-width: 768px) {
+      max-width: 100%;
+    }
+    @media (max-width: 480px) {
+      padding: 10px;
+    }
+  }
+}
+.main__drilling-work-inform-text {
+  width: 790px;
+  padding: 80px 0px 0px 0px;
+  @media (max-width: 1400px) {
+    text-align: center;
+    padding: 0;
+    width: 100%;
+  }
+}
+.main__drilling-work-inform-text2 {
+  padding: 80px 60px 0px 0px;
+  width: 790px;
+  @media (max-width: 1400px) {
+    padding: 0;
+    width: 100%;
+    text-align: center;
+  }
+}
+.main__drilling-work-inform-text-h2 {
+  @include default-h($text-dark-color);
+  padding-bottom: 15px;
+  @media (max-width: 1400px) {
+    padding: 0 40px 30px 40px;
+  }
+  @media (max-width: 864px) {
+    font-size: 40px;
+  }
+  @media (max-width: 576px) {
+    font-size: 30px;
+  }
+  @media (max-width: 480px) {
+    padding: 0 0 10px 0;
+    font-size: 25px;
+  }
+  @media (max-width: 360px) {
+    font-size: 20px;
+    padding: 0;
+  }
+  @media (max-width: 320px) {
+    font-size: 18px;
+    line-height: 35px;
+  }
+}
+.main__drilling-work-inform-text-p {
+  @include default-p($text-blue-color);
+  padding-bottom: 30px;
+  @media (max-width: 1400px) {
+    padding: 0 40px 30px 40px;
+  }
+  @media (max-width: 480px) {
+    display: none;
+  }
+}
+.main__drilling-work-inform-bt {
+  @media (max-width: 1400px) {
+    text-align: center;
+    padding-bottom: 40px;
+  }
+  @media (max-width: 480px) {
+    padding-bottom: 15px;
+  }
+  @media (max-width: 360px) {
+    padding-bottom: 5px;
+  }
+  .main__drilling-work-inform-bt-btn {
+    @include default-btn(237px, $text-light-color, $btn-link-dark-color, 0px);
+    @media (max-width: 480px) {
+      padding-block: 8px 8px;
+      max-width: 200px;
+      line-height: 20px;
+    }
+    @media (max-width: 360px) {
+      font-size: small;
+    }
+  }
+}
+.main__map-box {
+  @media (max-width: 480px) {
+    display: none;
+  }
+  .main__map {
+    position: relative;
+  }
+  .main__map-contacts {
+    margin-inline: auto;
+    background-color: $text-light-color;
+    font-family: $font;
+    max-width: 650px;
+    padding: 40px;
+    position: absolute;
+    top: 115px;
+    left: 160px;
+    z-index: 2;
+    @media (max-width: 1900px) {
+      max-width: 33%;
+      top: 100px;
+      left: 140px;
+    }
+    @media (max-width: 1400px) {
+      padding: 30px;
+      top: 25px;
+      left: 25px;
+    }
+    @media (max-width: 1024px) {
+      max-width: 50%;
+      top: 65px;
+    }
+    @media (max-width: 640px) {
+      max-width: 70%;
+    }
+    .main__map-contacts-all {
+      padding-bottom: 14px;
```

Опять-таки, не стоит задавать паддинги для нижних отступов, т.к. если бы у тебя был покрашен задний фон, то окраска продолжилась бы вместе с паддингом.
```suggestion
      margin-bottom: 14px;
```